torchtext/utils.py

```diff
@@ -73,7 +91,7 @@ def process_response(r):
         url = url + "&confirm=" + confirm_token
         response = session.get(url, stream=True)
 
-    process_response(response)
+    return process_response(response, root, filename)
 
 
 def unicode_csv_reader(unicode_csv_data, **kwargs):
```

Comment:
Since this API will be exposed to the outside, I would expect more docs and examples (I could add them).